packages/mdc-slider/adapter.js

```diff
@@ -51,12 +51,24 @@ class MDCSliderAdapter {
    */
   setThumbAttribute(name, value) {}
 
+  /**
+   * Removes attribute name on slider thumb Element to value
+   * @param {string} name
+   */
+  removeThumbAttribute(name) {}
+
   /**
    * Returns the bounding client rect for the slider Element
    * @return {?ClientRect}
    */
   computeBoundingRect() {}
 
+  /**
+   * Returns the tab index of the slider Element
+   * @return {number}
+   */
+  getTabIndex() {}
```

Comment:
Update this from the comment above ("getThumbTabIndex")

Reply:
👍